src/lib/OpenEXRCore/internal_dwa_compressor.h

```diff
@@ -845,13 +845,13 @@ DwaCompressor_uncompress (
     compressedDcBuf  = compressedAcBuf + (ptrdiff_t) (acCompressedSize);
     compressedRleBuf = compressedDcBuf + (ptrdiff_t) (dcCompressedSize);
 
-    if (compressedUnknownBuf >= dataPtrEnd ||
+    if (compressedUnknownBuf > dataPtrEnd ||
```

Comment:
These changes look unrelated to config consolidation.  Is this leaking in from your bounds checking?